typescript/sdk/src/token/types.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from 'zod';
+
+import {
+  TokenRouterConfigSchema,
+  WarpRouteDeployConfigSchema,
+} from './schemas.js';
+
+export type TokenRouterConfig = z.infer<typeof TokenRouterConfigSchema>;
+export type WarpRouteDeployConfig = z.infer<typeof WarpRouteDeployConfigSchema>;
```

Comment:
Nit: These probably don't need to be in a separate file from the schemas. I'd just put them together in types.ts

Reply:
I like the separation of concerns since the schemas can have its own business logic and get pretty big due to `refine()`